### ISSUE TITLE
Some questions about model trained by the RealBlur dataset 

### ISSUE THREAD

**Issue by kiyeonkimm** [swz30/MPRNet#70]
How many epochs did you train MPRNet through RealBlurJ and RealblurR respectively? 
Could you provide a checkpoint of the model trained with RealBlurJ and RealBlurR respectively?

Thanks!

**Comment by adityac8** [comment]
Hi

We train the model for 100 epochs. The pre-trained model and details can be found at https://github.com/swz30/MPRNet/issues/40#issuecomment-828429462

Thanks

**Comment by kiyeonkimm** [comment]
Thanks for the reply.

Are hyperparameters like learning rate and batch size and optimizer the same as default settings except for epoch and dataset configuration?

**Comment by adityac8** [comment]
Yes

**Comment by kiyeonkimm** [comment]
Thank you
